docs/extensibility/30-details-summary.md

```diff
@@ -21,6 +21,13 @@ This table lists the available parameters of the **data.details.header**, **data
 
 Extra parameters might be available for specific widgets.
 
+#### Status Header
+
+The **data.details.status.header** accepts one widget (for example [Badge](./50-list-and-details-widgets.md#badge)), displayed on the right side of the **Status** section header. This widget is used to summarize the status of the resource.
+
+<img src="./assets/display-widgets/StatusBadge.png" alt="Example of a status badge widget" width="40%" style="border: 1px solid #D2D5D9">
```

Comment:
Should this photo be displayed here or is it unnecessary?

Reply:
I'd drop it. It's more difficult to maintain screenshots, plus it's well-described, so it doesn't need one.